app/controllers/search_controller.rb

```diff
@@ -4,14 +4,24 @@ class SearchController < ApplicationController
 
   def index
     @query = query
-    @teams, @exact_team_exists = GroupSearch.new(@query).perform_search
-    @people, @person_match_exists = PersonSearch.new(@query).perform_search
+    @team_results = search_teams(@query)
```

Comment:
The @query is not needed due to it being an instance variable, it can be 
```ruby
  def index
    @query = query
    @team_results = search_teams
    @people_results = search_people
  end

  def search_teams
    search = GroupSearch.new(@query, SearchResults.new)
    search.perform_search
  end

  ...
```

or even, maybe(it will need testing...) 
```ruby
  def index
    @query = query
    @team_results = search(GroupSearch)
    @people_results = search(PersonSearch)
  end

  private

  def search(object)
    search = object.new(@query, SearchResults.new)
    search.perform_search
  end
```